README.md

```diff
@@ -88,25 +88,27 @@ to modify the build scripts. For example, to disable the local build cache when
+| buildCache.remote.setAllowInsecureProtocol | gradle.cache.remote.allowInsecureProtocol | GRADLE_CACHE_REMOTE_ALLOWINSECUREPROTOCOL |
+| buildCache.remote.setUrl                   | gradle.cache.remote.url                   | GRADLE_CACHE_REMOTE_URL                   |
+| buildCache.remote.setUrl                   | gradle.cache.remote.path                  | GRADLE_CACHE_REMOTE_PATH                  |
+| buildCache.remote.setUrl                   | gradle.cache.remote.shard                 | GRADLE_CACHE_REMOTE_SHARD                 |
 
 ### Gradle Enterprise Build Cache settings
 
-| Gradle Enterprise Build Cache API         | System property                          | Environment variable                     |
-|:------------------------------------------|:-----------------------------------------|:-----------------------------------------|
-| buildCache.remote.setEnabled              | gradle.cache.remote.enabled              | GRADLE_CACHE_REMOTE_ENABLED              |
-| buildCache.remote.setPush                 | gradle.cache.remote.push                 | GRADLE_CACHE_REMOTE_PUSH                 |
-| buildCache.remote.setAllowUntrustedServer | gradle.cache.remote.allowUntrustedServer | GRADLE_CACHE_REMOTE_ALLOWUNTRUSTEDSERVER |
-| buildCache.remote.setServer               | gradle.cache.remote.url                  | GRADLE_CACHE_REMOTE_URL                  |
-| buildCache.remote.setPath                 | gradle.cache.remote.path                 | GRADLE_CACHE_REMOTE_PATH                 |
-| buildCache.remote.setPath                 | gradle.cache.remote.shard                | GRADLE_CACHE_REMOTE_SHARD                |
+| Gradle Enterprise Build Cache API           | System property                              | Environment variable                       |
+|:--------------------------------------------|:---------------------------------------------|:-------------------------------------------|
+| buildCache.remote.setEnabled                | gradle.cache.remote.enabled                  | GRADLE_CACHE_REMOTE_ENABLED                |
+| buildCache.remote.setPush                   | gradle.cache.remote.push                     | GRADLE_CACHE_REMOTE_PUSH                   |
+| buildCache.remote.setAllowUntrustedServer   | gradle.cache.remote.allowUntrustedServer     | GRADLE_CACHE_REMOTE_ALLOWUNTRUSTEDSERVER   |
+| buildCache.remote.setAllowInsecureProtocol  | gradle.cache.remote.setAllowInsecureProtocol | GRADLE_CACHE_REMOTE_ALLOWINSECUREPROTOCOL  |
+| buildCache.remote.setServer/setPath         | gradle.cache.remote.url                      | GRADLE_CACHE_REMOTE_URL                    |
+| buildCache.remote.setPath                   | gradle.cache.remote.path                     | GRADLE_CACHE_REMOTE_PATH                   |
+| buildCache.remote.setPath                   | gradle.cache.remote.shard                    | GRADLE_CACHE_REMOTE_SHARD                  |
```

Comment:
It seems there are now competing properties, but the precedence isn't explained here.

Reply:
This is not new. It typically makes no sense to override more than one at a time.